debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (quilt)
```

Comment:
We do not need quilt. Quilt is used when patching upstream code as part of the deb build process